docker-compose.yml

```diff
@@ -120,34 +120,67 @@ services:
     # nodered opens ports on influxdb so it needs to be able to talk to it.
     links:
       - influxdb
+      - postfix
 
   influxdb:
     restart: unless-stopped
-    image: influxdb:1.2.4
+    image: influxdb:latest
     volumes:
       - "${TTN_DASHBOARD_DATA}influxdb:/var/lib/influxdb"
     environment:
       INFLUXDB_INIT_PWD: "${TTN_DASHBOARD_INFLUXDB_ADMIN_PASSWORD:-!notset}"
       PRE_CREATE_DB: "${TTN_DASHBOARD_INFLUXDB_INITIAL_DATABASE_NAME:-demo}"
+      INFLUXDB_BIND_ADDRESS: "influxdb:8088"
+
+  influxdb-backup:
+    restart: unless-stopped
+    build: influxdb-backup  
+    volumes:
+     # Dircectory for database backup inside "./data" 
+      - "${TTN_DASHBOARD_DATA}influx-backup:/var/lib/influxdb-backup"
+     # Connecting influxdb data directory to extra instances
+      - "${TTN_DASHBOARD_DATA}influxdb:/var/lib/influxdb"
+    environment:
+      INFLUX_HOST: influxdb                                  
+      S3_BUCKET_INFLUXDB: "${TTN_DASHBOARD_AWS_S3_BUCKET_INFLUXDB:-.}"
+      AWS_ACCESS_KEY_ID: "${TTN_DASHBOARD_AWS_ACCESS_KEY_ID:-.}"
+      AWS_SECRET_ACCESS_KEY: "${TTN_DASHBOARD_AWS_SECRET_ACCESS_KEY:-.}"
+      AWS_DEFAULT_REGION: "${TTN_DASHBOARD_AWS_DEFAULT_REGION:-.}"
+    links:
+      - influxdb
+
+
+  postfix:
+    restart: unless-stopped
+    build: 
+      context: ./postfix 
+      dockerfile: Dockerfile 
+      args:
+        relay_ip: "${TTN_DASHBOARD_MAIL_RELAY_IP:-.}"
+        host_name: "${TTN_DASHBOARD_MAIL_HOST_NAME:-.}"
+    ports:
+        - "${TTN_DASHBOARD_PORT_POSTFIX:-25}:25"
+
 
   grafana:
     restart: unless-stopped
-    image: grafana/grafana:4.3.2
+    image: grafana/grafana:latest
```

Comment:
see comment above about revisions. Perhaps `${TTN_DASHBOARD_GRAFANA_VERSION:-latest}`? (Then add comment in the variables section of the dockerfile.)